backend/endpoints/rooms.py

```diff
@@ -1,10 +1,12 @@
+from backend.schema.room import Role, RoomMember, RoomType
```

Comment:
Pleasee change the `from backend.schema.room`  to  just ` from schema.room`
this will cause the server to crash